pkg/kubelet/dockershim/exec.go

```diff
@@ -149,16 +152,28 @@ func (*NativeExecHandler) ExecInContainer(client libdocker.Interface, container
 
 	// Have to start this before the call to client.StartExec because client.StartExec is a blocking
```

Comment:
@ncdc - "Delay attempting to send a terminal resize" mentioned in the body of the PR